### ISSUE TITLE
Zoom event not working

### ISSUE THREAD

**Issue by herlon214** [KoRiGaN/Vue2Leaflet#47]
When I do a zoom in the map the event isn't being called. My component:

```
<template>
  <v-map ref="map" :zoom="zoom" :center="center" v-on:l-zoom="zoom">
    <v-tilelayer :url="titleLayer"></v-tilelayer>
    <v-marker :lat-lng="center"></v-marker>
  </v-map>
</template>

<script>

// Component
export default {
  name: 'map',

  data () {
    return {
      titleLayer: 'secret'
    }
  },

  methods: {

    zoom (evt) {
      console.log(evt) // not showing anything
    },
  },

  computed: {
    center () {
      return this.$store.state.map.center
    },
    zoom () {
      return this.$store.state.map.zoom
    }
  }
}
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped>
</style>

```

**Comment by ais-one** [comment]
You should use **v-on:l-moveend** instead of **v-on:l-zoom**

There is no zoom event handled by vue2-leaflet

**Comment by KoRiGaN** [comment]
v-on:l-zoom implemented in v0.0.50
